public/app/features/alerting/unified/Insights.tsx

```diff
@@ -0,0 +1,80 @@
+import { css } from '@emotion/css';
+import React from 'react';
+
+import { GrafanaTheme2 } from '@grafana/data';
+import { EmbeddedScene, PanelBuilders, SceneFlexItem, SceneFlexLayout, SceneQueryRunner } from '@grafana/scenes';
+import { useStyles2 } from '@grafana/ui';
+
+function getScene() {
+  const queryRunner1 = new SceneQueryRunner({
+    datasource: {
+      type: 'loki',
+      //since all cloud instances have a provisioned alert state history data source,
+      //we should be able to use its uid here
+      uid: 'CKA1_2hVz',
+    },
+    queries: [
+      {
+        refId: 'A',
+        expr: '{from="state-history"} | json | current="Alerting"',
+      },
+    ],
+  });
+
+  const queryRunner2 = new SceneQueryRunner({
+    datasource: {
+      type: 'prometheus',
+      uid: 'gdev-prometheus',
+    },
+    queries: [
+      {
+        refId: 'A',
+        expr: 'avg by (job, instance, mode) (rate(node_cpu_seconds_total[5m]))',
+      },
+    ],
+  });
```

Comment:
This is just to show how to display several panels but this data is not relevant